Tools/WinMLRunner/src/OutputHelper.h

```diff
@@ -278,7 +284,17 @@ class OutputHelper
             std::cout << std::endl;
         }
         std::cout << "  Working Set Memory usage (evaluate): " << firstEvalMemoryUsage << " MB" << std::endl;
+
+        if (isPerformanceConsoleOutputVerbose)
+        {
+            std::cout << "  Peak Working Set Memory Difference (from start to load): " << firstLoadPeakWorkingSetUsage << " MB" << std::endl;
+            std::cout << "  Peak Working Set Memory Difference (from model load to session creation): " << firstSessionPeakWorkingSetUsage << " MB" << std::endl;
+            std::cout << "  Peak Working Set Memory Difference (from session to bind): " << firstBindPeakMemoryUsage << " MB" << std::endl;
+            std::cout << "  Peak Working Set Memory Difference (from bind to evaluate): " << firstEvalPeakMemoryUsage << " MB" << std::endl;
+        }
+
         std::cout << "  Working Set Memory usage (load, bind, session creation, and evaluate): " << firstIterationWorkingSetMemoryUsage << " MB" << std::endl;
+        std::cout << "  Peak Working Set Memory Difference (load, bind, session creation, and evaluate): " << firstIterationPeakWorkingSet << " MB" << std::endl;
```

Comment:
Can we keep Working Set Memory Usage and Peak Working Set Memory Usage together for the (load, bind, session creation, and evaluate) metric?

Reply:
From  Working Set Memory usage (load): 9.54297 MB
  Working Set Memory usage (session creation): 12.1602 MB
  Working Set Memory usage (bind): 1.09375 MB
  Working Set Memory usage (evaluate): 1.26563 MB
  Peak Working Set Memory Difference (from start to load): 9.21094 MB
  Peak Working Set Memory Difference (from model load to session creation): 15.4883 MB
  Peak Working Set Memory Difference (from session to bind): 0 MB
  Peak Working Set Memory Difference (from bind to evaluate): 0.0820313 MB
  Working Set Memory usage (load, bind, session creation, and evaluate): 24.0625 MB
  Peak Working Set Memory Difference (load, bind, session creation, and evaluate): 24.7813 MB

to

 Working Set Memory usage (load): 9.54297 MB
  Working Set Memory usage (session creation): 12.1602 MB
  Working Set Memory usage (bind): 1.09375 MB
  Working Set Memory usage (evaluate): 1.26563 MB
  Working Set Memory usage (load, bind, session creation, and evaluate): 24.0625 MB
  
Peak Working Set Memory Difference (from start to load): 9.21094 MB
  Peak Working Set Memory Difference (from model load to session creation): 15.4883 MB
  Peak Working Set Memory Difference (from session to bind): 0 MB
  Peak Working Set Memory Difference (from bind to evaluate): 0.0820313 MB
  Peak Working Set Memory Difference (load, bind, session creation, and evaluate): 24.7813 MB